downstream/modules/platform/ref-access-control-requirements.adoc

```diff
@@ -0,0 +1,176 @@
+
+[id="ref-access-control-requirements_{context}"]
+
+= Access control requirements
+
+[role="_abstract"]
+
+{PlatformName} uses a number of ports to communicate with its services. These ports must be open and available for incoming connection to the {PlatformName} server in order for it to work. Ensure that these ports are available and are not being blocked by a firewall.
+
+The following tables provide the default {PlatformName} destination ports required for each application.
+
+.PostgreSQL
+[options="header"]
+|===
+|Port |Protocol |Service |Required for
+|22
+|TCP
+|SSH
+|Remote access during installation
+|5432
+|TCP
+|Postgres
+a|Default port
+
+ALLOW connections from controller(s) to database port
+|===
+
+.{ControllerNameStart}
+[options="header"]
+|===
+|Port |Protocol |Service |Required for
+|22
+|TCP
+|SSH
+|Installation
+
+|80
+|TCP
+|HTTP
+|UI/API & {ExecEnvShort} pulls
```

Comment:
What does "execution environment pulls" mean ?

Reply:
Hi @Spredzy I copied this information, word for word from this [google doc](https://docs.google.com/document/d/1g8oIX5xiZEEvoTmjB1B1REzSCw55aJpNpBSBd4VGv6g/edit#bookmark=id.69cn3hp5zhlf), provided in [AAP-1439](https://issues.redhat.com/browse/AAP-1439) that was created by Phil Griffiths. 

That doc seems to be a collaboration effort from different SMEs. I've had difficulty pinning down an SME who contributed to that doc to tell me what some of those comments mean. Should I remove "execution environment pulls" from the table?

Reply:
@Spredzy I've received an explanation of "EE pulls." =  "Pulls execution environments from your chosen container registry" and made the edit.